### PR TITLE
TEC-383: List-style views > Avoid "Now Onwards" heading 

### PR DESCRIPTION
**Pull request by mitogh**
Update the logic to render the upcoming label for the views using the current view uses a List type of view.

Artifact: https://d.pr/v/wBMaqn

Ticket: TEC-383

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -223,6 +223,7 @@ Remember to always make a backup of your database and files before updating!
 
 = [5.6.1] TBD =
 
+* Tweak - Adjust wording for 'onwards' for different scenarios on views using the List type of view [TEC-3831]
```
Comment (borkweb) [inline readme.txt:226]:
```suggestion
* Tweak - Reduced the usage of the word "onwards" on list-style view date range headings where simpler headings are better suited. [TEC-3831]
```

**Review by borkweb — CHANGES_REQUESTED** [review]
Just a few minor text changes and then this is good to go once tests pass!

File: src/Tribe/Views/V2/Views/Traits/List_Behavior.php [hunk 2]
```diff
@@ -134,6 +132,7 @@ static function ( $event )
 			            && $start->format( 'Y-m-d' ) !== $end->format( 'Y-m-d' )
 		            );
 
+
```
Comment (Camwyn) [inline src/Tribe/Views/V2/Views/Traits/List_Behavior.php:135]:
```suggestion
```
Superfluous line?

**Review by borkweb — DISMISSED** [review]
:+1: - after tests pass, at least.

**Pull request merged**